packages/flutter_tools/templates/plugin/.idea/workspace.xml.tmpl

```diff
@@ -2,7 +2,7 @@
 <project version="4">
   <component name="FileEditorManager">
     <leaf>
-      <file leaf-file-name="main.dart" pinned="false" current-in-tab="true">
+      <file leaf-file-name="{{projectName}}.dart" pinned="false" current-in-tab="true">
```

Comment:
Isn't the file still called main.dart in any project from the app template?

Reply:
Yes, but this is for plugins. See below for discussion.